governance.md

```diff
@@ -17,4 +17,16 @@ https://github.com/pangeo-data/governance
 
 ## Institutional Partners and Funding
 
+The Steering Council is the primary leadership for the project. No outside institution, individual or legal entity has the ability to own, control, usurp or influence the project other than by participating in the Project as Contributors and Council Members. However, because institutions are the primary funding mechanism for the project, it is important to formally acknowledge institutional participation in the project. These are Institutional Partners.
+
+An Institutional Contributor is any individual Project Contributor who contributes to the project as part of their official duties at an Institutional Partner. Likewise, an Institutional Council Member is any Project Steering Council Member who contributes to the project as part of their official duties at an Institutional Partner.
+
+With these definitions, an Institutional Partner is any recognized legal entity anywhere in the world that employs at least one Institutional Contributor or Institutional Council Member. Institutional Partners can be for-profit or non-profit entities.
+
+Institutions become eligible to become an Institutional Partner by employing individuals who actively contribute to The Project as part of their official duties. To state this another way, the only way for an Institutional Partner to influence the project is by actively contributing to the open development of the project, on equal terms with any other member of the community of Contributors and Council Members. Merely using Project Software or Services in an institutional context does not allow an entity to become an Institutional Partner. Financial gifts do not enable an entity to become an Institutional Partner. Once an institution becomes eligible for Institutional Partnership, the Steering Council must nominate and approve the Partnership.
+
+If an existing Institutional Partner no longer has a contributing employee, they will be given a one-year grace period for other employees to begin contributing.
+
+An Institutional Partner is free to pursue funding for their work on The Project through any legal means. This could involve a non-profit organization raising money from private foundations and donors or a for-profit company building proprietary products and services that leverage Project Software and Services. Funding acquired by Institutional Partners to work on The Project is called Institutional Funding. However, no funding obtained by an Institutional Partner can override the decisions of the Steering Council. If a Partner has funding to do Project work and the Council decides to not pursue that work as a project, the Partner is free to pursue it on their own. However, in this situation, that part of the Partner’s work will not be under the Project umbrella.
```

Comment:
Q: should we suggest that Institutional Partners are encouraged to notify the steering committee of pending funding proposals?

Reply:
Yes. And even better, to consult with the project before submitting proposals, to ensure that the work proposed is in line with the overall project vision.